tools/ci_build/amd_hipify.py

```diff
@@ -311,6 +307,12 @@ def hipify(src_file_path, dst_file_path):
     s = s.replace("hipdnn", "miopen")
     s = s.replace("HIPDNN_STATUS_SUCCESS", "miopenStatusSuccess")
     s = s.replace("HIPDNN", "MIOPEN")
+    # other MIOpen compatibility problems
+    s = s.replace("miopenSetPoolingNdDescriptor", "CompactMiopenSetNdPoolingDescriptor")
+    s = s.replace("miopenPoolingForward", "CompactMiopenPoolingForward")
+    s = s.replace("MIOPEN_POOLING_MAX", "miopenPoolingMax")
```

Comment:
Is it possible we add PoolingForward and SetNdPoolingDescriptor to both cudnn_common.h and miopen_common.h with different impls so that we don't need these specific string replacements here?

Reply:
Some of them is possible, but for constants like `MIOPEN_POOLING_MAX` we have to keep the replacement due to the bugs in the `hipify` tool (`MIOPEN_POOLING_MAX` doesn't exist at all, and `miopenPoolingMax` is the right constant)